assignments/callbacks.js

```diff
@@ -24,25 +24,76 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+    return cb(arr);
 }
 
+function arrLength(length) {
+    console.log(length.length);
+}
+
+getLength(items, arrLength);
```

Comment:
This is another example of why it would be useful to have meaningful parameter names - what is a `length.length`? You're imputing an array here, consider changing this for future projects